### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_VS Code FAQ_Resize Text: At 150-200% zooming Download link is not visible to the user

### ISSUE THREAD

**Issue by Amlan454** [microsoft/vscode-docs#2645]
Check out Keros!- Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details:**
code.visualstudio.com/docs/supporting/faq
Browser: Edge, chrome

**Note:
This issue is observed through out the application:**
code.visualstudio.com/docs
code.visualstudio.com/docs/supporting/faq
code.visualstudio.com/docs/setup/linux
code.visualstudio.com/docs/getstarted/settings
code.visualstudio.com/docs/python/python-tutorial
code.visualstudio.com/updates/v1_30
code.visualstudio.com/docs/editor/whyvscode
code.visualstudio.com/docs/getstarted/tips-and-tricks
code.visualstudio.com/updates/v1_29
code.visualstudio.com/docs/editor/integrated-terminal
code.visualstudio.com/docs/editor/debugging
code.visualstudio.com/docs/editor/versioncontrol

**Repro Steps:**
1)Hit the URL(code.visualstudio.com/docs/supporting/faq)and login by using appropriate credential
2)Zooming from 150-200% and check whether Download link is visible or not

**Actual Result:**
At 150-200% zooming Download link is not visible to the user

**Expected Result:**
Up to 200% zooming each and every control should be visible to the user

**User Impact:**
Low vision users will face difficulty to find out the control when they are navigating the page in magnifier mode

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

**Comment by Amlan454** [comment]
[Bug_2645_Download link is not visible at 150-200% zooming.pptx](https://github.com/microsoft/vscode-docs/files/3161732/Bug_2645_Download.link.is.not.visible.at.150-200.zooming.pptx)

**Comment by Amlan454** [comment]
![Bug_2645](https://user-images.githubusercontent.com/42638683/57694322-093ac180-7669-11e9-982d-0efbb94014db.png)

**Comment by auchenberg** [comment]
Same issue here @misolori. Our header design simply don't work with zooms.

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
